### PR TITLE
#1-4Task. sofa.tatarinova

### PR DESCRIPTION
**Pull request by sofa-tata**
https://travis-ci.org/github/sofa-tata/js-assignments

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -114,7 +115,11 @@ function removeLeadingAndTrailingWhitespaces(value) {
  *   'cat', 3 => 'catcatcat'
  */
 function repeatString(value, count) {
-    throw new Error('Not implemented');
+    let result = '';
+    for (let i = 0; i < count; i += 1){
+        result += value;
+    }
+    return result;
```
Comment (vramaniuk) [inline task/01-strings-tasks.js]: Please, try to solve task using
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

File: task/01-strings-tasks.js [hunk 2]
```diff
@@ -145,7 +150,7 @@ function removeFirstOccurrences(str, value) {
  *   '<a>' => 'a'
  */
 function unbracketTag(str) {
-    throw new Error('Not implemented');
+    return str.slice(1, str.length - 1);
```
Comment (vramaniuk) [inline task/01-strings-tasks.js]: return str.slice(1, - 1); It also works. It's more consistent

File: task/04-arrays-tasks.js [hunk 3]
```diff
@@ -175,7 +175,7 @@ function getHead(arr, n) {
  *    [ 'a', 'b', 'c', 'd'], 3  => [ 'b', 'c', 'd' ]
  */
 function getTail(arr, n) {
-   throw new Error('Not implemented');
+   return arr.slice(arr.length - n);
```
Comment (vramaniuk) [inline task/04-arrays-tasks.js]: return arr.slice(-n);   This is better

File: task/04-arrays-tasks.js [hunk 4]
```diff
@@ -200,7 +200,7 @@ function getTail(arr, n) {
  *    +'30,31,32,33,34'
  */
 function toCsvText(arr) {
-   throw new Error('Not implemented');
+   return arr.map((v) => v.join(',')).join('\n');
```
Comment (vramaniuk) [inline task/04-arrays-tasks.js]: `return arr.join('\n');` works

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Please, privide the direct link to your task success travis build

File: task/01-strings-tasks.js (hunk 1 above)
Comment (sofa-tata) [reply task/01-strings-tasks.js]: Okay, thanks

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**